src/e3/anod/spec.py

```diff
@@ -245,9 +245,10 @@ def __init__(
         # Create the QualifiersManager.
         # Skip if the name generator is disabled.
         if self.enable_name_generator:
```

Comment:
I think you should use `if self.enable_name_generator and self.kind != "source"`

Reply:
Done